tests/Integration/WorkloadManagementSystem/Test_Client_WMS.py

```diff
@@ -633,12 +632,12 @@ def test_JobManagerClient_removeJob():
 
     # Wait for the optimizers to run
     startingTime = time.time()
-    while time.time() - startingTime < 5 * 60:
+    while time.time() - startingTime < 7 * 60:
```

Comment:
Waiting 21 minutes for CI to run one test isn't really practical (though it's not much worse than the existing 15 minutes).